### PR TITLE
Decode Morse Code

### PR DESCRIPTION
**Pull request by yusufobr**
## Decode Morse Code 👨‍💻 👨‍💻
In this milestone, 🗿 we managed to implement the following:

-  Set up Linters. 
-  Decode the morse code 
-  Followed ruby best practices 
-  Corrected linter errors

### CODE REVIEW

File: decoder.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  end
+  "#{word} "
+end
+
+INPUT = '.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...'.freeze
+output = ''
+INPUT.split('   ').each do |s|
+  output += decode(s)
+end
+puts output
```
Comment (abdulhamiid) [inline decoder.rb]: Please, let adhere to the project core requirements as much as possible, According to the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md) please implement the following 😊

- Create a method to decode a Morse code character
- Create a method to decode an entire word in Morse code
- Create a method to decode the entire message in Morse code
- Reuse methods whenever possible

Running the lines of code below from the terminal

`p decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")`

❗ Result got 
image

Expected (space between words)
`"A BOX FULL OF RUBIES"`

Please correct this. 🛠👍

File: README.md [hunk 2]
```diff
@@ -1 +1,131 @@
...
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Aamir Asaram**
+
+- GitHub: [@aamir-asaram](https://github.com/aamir-asaram)
+- Twitter: [@aaaaamir\_\_](https://twitter.com/aaaaamir__)
+- LinkedIn: [Aamir Asaram](https://www.linkedin.com/in/aamir-asaram/)
```
Comment (abdulhamiid) [inline README.md]: Please fix the broken `Twitter` link, let's avoid broken links in our readme for a professional repository. 🤗

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi Team 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Run a program using the command line.
- ✔️ Use Ruby syntax for basic programming operations.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by addod19** [comment]
## Hi @yusufobr and @aamir-asaram, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because linters are failing

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by addod19 — APPROVED** [review]
## Hi @yusufobr and @aamir-asaram, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [x] Nice job fixing the code and link in README file :rocket: 
Screenshot from 2023-08-24 03-26-24



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aamir-asaram** [comment]
Thank you for the helpful reviews @addod19 @abdulhamiid 😄

**Pull request merged**